src/live-common-setup.js

```diff
@@ -22,6 +22,7 @@ setPlatformVersion("0.0.1");
 setSupportedCurrencies([
   "bitcoin",
   "ethereum",
+  "elrond",
```

Comment:
As discussed can you please remove this for now?

This will allow to merge silently and the integration can still be tested using the env var: `EXPERIMENTAL_CURRENCIES=elrond`